web/static/js/components/idea_submission_form.jsx

```diff
@@ -73,7 +89,7 @@ class IdeaSubmissionForm extends Component {
                 ref={input => { this.ideaInput = input }}
                 value={this.state.body}
                 onChange={this.handleIdeaChange}
-                placeholder="we're actively trying to improve"
+                placeholder={PLACEHOLDER_PREFIX + ' ' + PLACEHOLDER_TEXTS[this.state.category]}
```

Comment:
2 things here. 
1. just inline the string `"Ex. " rather than declaring a whole new constant up above
1. use string interpolation with template literals. pretty sure the linter should barf with the concatenation